src/controllers.php

```diff
@@ -2,25 +2,49 @@
 
 use Symfony\Component\HttpFoundation\Request;
 use Symfony\Component\HttpFoundation\Response;
-use Symfony\Component\HttpFoundation\JsonResponse;
-use Symfony\Component\HttpFoundation\RedirectResponse;
-use Symfony\Component\HttpKernel\Exception\NotFoundHttpException;
-
-//Request::setTrustedProxies(array('127.0.0.1'));
+use Symfony\Component\HttpFoundation\Session\Session;
+use Silex\Provider\FormServiceProvider;
 
 $app->get('/', function () use ($app) {
-    return $app['twig']->render('index.html.twig');
-})
+
+    if (($app['session']->get('is_logged')) === 1) {
+            return $app['twig']->render('index.html.twig');
+    } else {
+            return $app->redirect('/login');
```

Comment:
bledne wciecie

Reply:
Nie rozumiem. Jedyne co cs podkreśla to że otwierający nawias dla wywołania funkcji musi być ostatni w linii - ale jest ostatni. Nie ma po niem żadnej spacji.

A na czym polega błędność wcięcia? Inne wcięcia niż to obecnie są podkreślane przez cs jako błędne.

Reply:
przed return jest osiem spacji a nie 4

Reply:
Bo tego się domaga cs :P

Reply:
wyswietl komunikat jaki generuje ci CS podaj komende jaka uzywasz do sprawdzenia CS a najlepiej skonfiguruj travisa dla tego repo